project/Dependencies.scala

```diff
@@ -17,37 +17,38 @@
+  val scalacheck          = "org.scalacheck"             %% "scalacheck"               % "1.13.5"
+  val jts                 = "com.vividsolutions"          %  "jts-core"                % "1.14.0"
 
-  val monocleCore   = "com.github.julien-truffaut" %% "monocle-core"  % Version.monocle
-  val monocleMacro  = "com.github.julien-truffaut" %% "monocle-macro" % Version.monocle
+  val monocleCore         = "com.github.julien-truffaut" %% "monocle-core"             % Version.monocle
+  val monocleMacro        = "com.github.julien-truffaut" %% "monocle-macro"            % Version.monocle
 
-  val openCSV       = "com.opencsv" % "opencsv" % "3.9"
+  val openCSV             = "com.opencsv"                 % "opencsv"                  % "3.9"
 
-  val spire         = "org.spire-math" %% "spire" % "0.13.0"
+  val spire               = "org.spire-math"             %% "spire"                    % "0.13.0"
 
-  val sprayJson     = "io.spray" %% "spray-json" % Version.sprayJson
+  val sprayJson           = "io.spray"                   %% "spray-json"               % Version.sprayJson
 
-  val apacheMath    = "org.apache.commons" % "commons-math3" % "3.6.1"
+  val apacheMath          = "org.apache.commons"          % "commons-math3"            % "3.6.1"
 
-  val chronoscala   = "jp.ne.opt" %% "chronoscala" % "0.1.3"
+  val chronoscala         = "jp.ne.opt"                  %% "chronoscala"              % "0.1.3"
 
-  val awsSdkS3      = "com.amazonaws" % "aws-java-sdk-s3" % "1.11.143"
+  val awsSdkS3            = "com.amazonaws"               % "aws-java-sdk-s3"          % "1.11.143"
 
-  val scalazStream  = "org.scalaz.stream" %% "scalaz-stream" % "0.8.6a"
+  val scalazStream        = "org.scalaz.stream"          %% "scalaz-stream"            % "0.8.6a"
 
-  val sparkCore     = "org.apache.spark" %% "spark-core" % Version.spark
-  val hadoopClient  = "org.apache.hadoop" % "hadoop-client" % Version.hadoop
+  val sparkCore           = "org.apache.spark"           %% "spark-core"               % Version.spark
+  val hadoopClient        = "org.apache.hadoop"           % "hadoop-client"            % Version.hadoop
 
-  val avro          = "org.apache.avro" % "avro" % "1.8.2"
+  val avro                = "org.apache.avro"             % "avro"                     % "1.8.2"
 
-  val slickPG      = "com.github.tminglei" %% "slick-pg" % "0.15.0"
+  val slickPG             = "com.github.tminglei"        %% "slick-pg"                 % "0.15.0"
 
-  val parserCombinators = "org.scala-lang.modules" %% "scala-parser-combinators" % "1.0.6"
+  val parserCombinators   = "org.scala-lang.modules"     %% "scala-parser-combinators" % "1.0.6"
 
-  val jsonSchemaValidator = "com.networknt" % "json-schema-validator" % "0.1.7"
+  val jsonSchemaValidator = "com.networknt"               % "json-schema-validator"    % "0.1.7"
+  val scaffeine           = "com.github.blemale"         %% "scaffeine"                % "2.1.0"
```

Comment:
Are we able to use version [`2.2.0`](http://search.maven.org/#artifactdetails%7Ccom.github.blemale%7Cscaffeine_2.11%7C2.2.0%7Cjar).
A CQ has already been filed and approved for that version.